### ISSUE TITLE
Any plans for this mod improvement?

### ISSUE THREAD

**Issue by Yiddan** [DumpsterDave/FS22_PriceWatcher#3]
Hello dave :)
The mod works flawlessly as it is atm, i really like it.
Im guessing if you consider it "done" now, or have any plans for its future?

I personally thought about a few suggestions, that could improve my own experience with it.
I get the plan is certainly to keep pricewatcher as simple as possible, and im not sure how much my suggestions could complicate it...
So maybe its still best to share my ideas, If you dont mind.

- I'd like settings to be saved per-save, so i could only track goods values i need on each of my playthroughs
- A simple ingame "customization UI" could be of help, with checkboxes to toggle on/off tracking for all goods (but im comfortable with editing the config file by hand, as its not something me or i guess a player would change a lot)
- A way to print (on screen, wherever you could do it) max sell values actually registered, and relative month. I have hard time remembering all products max sell prices and currently rely on a text file on my desktop, efficient but ingame info would be neat imo)

**Comment by DumpsterDave** [comment]
Yes, there are more features to come.  I do plan on adding some UI elements in the future.  I plan to make the tracked notifications static so that it shows the current price, max annual price, and max ever price along with trend and sell station so that you can easily glance at any time to see the data. 

The per-save config should be easy enough to implement.  I'll try and implement that in my next release.

I've also thought about a similar UI to control the tracking of goods as well.

Good suggestions, thank you!

**Comment by Yiddan** [comment]
Oh i like your plan to implement a "static" ui with all current/max/alwaysmax/month !
Glad to read you have ideas for the future of this mod, and you're very welcome! Thanks to YOU for your work ;)
